nuntium/templates/nuntium/profiles/welcome.html

```diff
@@ -4,9 +4,107 @@
 {% block content %}
 
 <div class="page-header">
-    <h2>{% trans "Welcome" %}</h2>
+    <h2>
+       {% blocktrans with name=writeitinstance.name %}
+         Welcome! This is {{ name }}
+       {% endblocktrans %}
+    </h2>
 </div>
 
-<p>{% trans "Write to your representative etc." %}</p>
+<p>
+  {% blocktrans with name=writeitinstance.name %}
+    This is your &ldquo;{{name}}&rdquo; message-sending site.
+  {% endblocktrans %}
+</p>
+{% if writeitinstance.config.testing_mode %}
+  <p>
+    {% blocktrans %}
+      You're currently running in <strong>testing mode</strong>, so all
+      the emails sent will go to you (and not the recipients).
+      Contact us if you want this restriction lifted and you're ready to
+      send real messages to real people!
+    {% endblocktrans %}
+  </p>
+{% endif %}
+<p>
+  {% if writeitinstance.persons.count %}
+    {% blocktrans count qty_pers=writeitinstance.persons.count %}
+      You've got only one recipient in your database.
+    {% plural %}
+      You've got {{ qty_pers }} recipients in your database.
+    {% endblocktrans %}
+  {% else %}
+    {% blocktrans %}
+      <strong>There aren't any recipients in your database</strong>
+      &mdash; you need to get some in there before anyone can send any messages!
+      <br>
+      Has your data source got recipients in it? Has it loaded OK?
```

Comment:
Surely we can tell them this? i.e. "Your data source still hasn't finished loading" vs "We loaded your data source, but couldn't find any recipients in it"?